### DISCUSSION TITLE
Remembering the last color annotation

### DISCUSSION THREAD

**Discussion by solisgb** [sumatrapdfreader/sumatrapdf#4296] [Ideas]
When creating an annotation, a default color is currently pre-selected, which can be changed manually. My request: If a user selects a different color for an annotation, the next annotation should default to the last used color instead of the original default.

**Comment by GitHubRulesOK** [comment]
AFAIK it does but it needs to be done in the settings file see my set Highlight to red in the comment https://github.com/sumatrapdfreader/sumatrapdf/discussions/4293#discussioncomment-9627590

closed and reopened with same advanced setting that is now the default
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6876126f-e48f-43e6-8b71-9334d8949174)

What you seem to be asking is when user changes one in editor that it become the default which is a different slant that may have been asked in an issue ?

**Comment by solisgb** [comment]
It's not a problem. Perhaps it would be useful if the annotation color
selection could be done from the editor, without changing the configuration
file. Best regards
